docs/source/generics.rst

```diff
@@ -701,9 +680,29 @@ When the decorator alters the signature, parameter specifications truly show the
         return value + 42
 
     a = add_forty_two(3)
-    reveal_type(a)  # str
-    foo('x')    # Type check error: incompatible type "str"; expected "int"
+    reveal_type(a)      # Revealed type is "builtins.str"
+    add_forty_two('x')  # error: Argument 1 to "add_forty_two" has incompatible type "str"; expected "int"
 
+Or insert an argument:
+
+.. code-block:: python
+
+    from typing import Callable, Concatenate, ParamSpec, TypeVar
```

Comment:
Brought this up in #14584, but maybe we actually should use `typing_extensions` here for Concatenate and ParamSpec, since it will work on all supported Python versions.